docs/en/running-node/open-node-tor-onion.md

```diff
@@ -83,9 +83,9 @@ Feel free to adjust above to your preferred conventions, just remember to adjust
 
 [Download](/interacting/download-monero-binaries/) and [verify](/interacting/verify-monero-binaries/) the file.
 
-Extract `tar -xf monero-linux-x64-v0.17.1.9.tar.bz2` (adjust filename).
+Extract `tar -xf monero-*.tar.bz2` (adjust filename if necessary).
 
-Move binaries to `/opt/monero/` with `mv monero-x86_64-linux-gnu-v0.17.1.9/* /opt/monero/` then `chown -R monero:monero /opt/monero`
+Move binaries to `/opt/monero/` with `sudo mv monero-*/* /opt/monero/` then `sudo chown -R monero:monero /opt/monero`
```

Comment:
have to be careful here and consider other possible folders in the dir monero-docs/* monero-gui/* monero-lws/* monero-python/* monero-site/*

Reply:
would you like to use a {{ macro }} here for current monero version instead?